Github workflow cache keeps crashing

This Github workflow cache is causing regular problems and fails the following message:

```
##[debug]Evaluating condition for step: '📦 Install FlowNet'
##[debug]Evaluating: success()
##[debug]Evaluating success:
##[debug]=> true
##[debug]Result: true
##[debug]Starting: 📦 Install FlowNet
##[debug]Loading inputs
##[debug]Loading env
Run source $VENV_PATH/bin/activate
##[debug]/bin/bash -e /home/runner/work/_temp/a491226b-12e9-4f98-8cdf-f32e4e22297e.sh
/home/runner/work/_temp/a491226b-12e9-4f98-8cdf-f32e4e22297e.sh: line 1: /home/runner/work/flownet/flownet/flownet_venv/bin/activate: No such file or directory
##[error]Process completed with exit code 1.
##[debug]Finishing: 📦 Install FlowNet
```

--- a/.github/workflows/flownet.yml
+++ b/.github/workflows/flownet.yml
@@ -34,20 +34,10 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
-      - name: 📁 Cache virtual environment
-        id: cache-venv
-        uses: actions/cache@v2
-        with:
-          path: ${{ env.VENV_PATH }}
-          # As long as this issue is open https://github.com/actions/cache/issues/2#issuecomment-673493515
-          # we add a "secret" manual cache ID that can be changed in the GitHub UI when necessary.
-          key: ${{ secrets.manual_cache_id }}-${{ runner.os }}-${{ matrix.python-version }}-${{ hashFiles('build_environment.sh') }}-${{ hashFiles('setup.py') }}
-
       - name: 📦 Install non-python dependencies
         run: sudo bash ./apt_install.sh
 
       - name: 🏗️ Build Python environment
-        if: steps.cache-venv.outputs.cache-hit != 'true'
         run: bash ./build_environment.sh $VENV_PATH /usr/bin/flow
 
       - name: 🍿 Download FlowNet test data

